src/levanter/models/llama.py

```diff
@@ -0,0 +1,404 @@
+from dataclasses import dataclass
+from typing import Callable, Optional, Tuple
+
+import equinox as eqx
+import jax
+import jax.numpy as jnp
+import jax.random as jrandom
+
+import haliax as hax
+import haliax.nn as hnn
+from haliax import Axis, NamedArray
+from haliax.jax_utils import named_call
+from haliax.nn.scan import Stacked
+
+from levanter.compat.torch_serialization import StateDictSerializationMixin
+from levanter.models.gpt2 import ACT2FN
+from levanter.models.lm_model import LmConfig
+
+
+@LmConfig.register_subclass("llama")
+@dataclass(frozen=True)
+class LlamaConfig:
+    """Config for LlamaModel
+
+    Args:
+        seq_len (int, optional): maximum length of the input sequence. Defaults to 2048.
```

Comment:
I believe max_position_embeddings is the maximum length of the input sequence in the LLamaConfig.